sdk/resourcemanagerhybrid/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/arm/CountryIsoCode.java

```diff
@@ -512,11 +512,7 @@
     /**
      * Country ISO code {@code MO}.
      */
-    public static final CountryIsoCode MACAO = fromString("MO");
-    /**
-     * Country ISO code {@code MK}.
-     */
-    public static final CountryIsoCode MACEDONIA = fromString("MK");
```

Comment:
This is a public API so we need to verify whether or not it has shipped before we make this change.